### ISSUE TITLE
Connection to small Bamboo Folio isn't working anymore

### ISSUE THREAD

**Issue by muesli** [tuhiproject/tuhi#137]
I used to get it to work with my small Bamboo Folio, but with latest git, I can seemingly connect to it, but not retrieve anything:

Output from tuhi:
```
02:52:13 INFO: tuhi.ble.E5:DF:80:AE:XX:YY: Disconnected
02:52:15 INFO: tuhi.ble.E5:DF:80:AE:XX:YY: Connecting
02:52:16 INFO: tuhi.ble.E5:DF:80:AE:XX:YY: Connection established
02:52:17 INFO: tuhi.wacom: Press the button now to confirm
02:52:19 ERROR: tuhi.wacom: device time is out by more than 5 minutes
02:52:19 INFO: tuhi.wacom: device name is b'muesli Folio\n'
02:52:19 INFO: tuhi.wacom: firmware is B160929A-C161130A
02:52:19 INFO: tuhi.wacom: registration completed
02:52:19 INFO: tuhi.ble.E5:DF:80:AE:XX:YY: Disconnecting
02:52:22 INFO: tuhi.ble.E5:DF:80:AE:XX:YY: Disconnected
```

Output from tuhi-kete:
```
tuhi> search
INFO: Search started
INFO: Unregistered device: E5:DF:80:AE:XX:YY - muesli Folio
tuhi> register E5:DF:80:AE:XX:YY
INFO: E5:DF:80:AE:XX:YY - muesli Folio: Press button on device now
INFO: E5:DF:80:AE:XX:YY - muesli Folio: Registration successful
tuhi> info
E5:DF:80:AE:XX:YY - muesli Folio
        Battery level: 0%, unknown
        Available drawings:
```

**Comment by whot** [comment]
based on that output, all is fine except that there aren't any drawings available at this point. Once you draw a few things, start `listen` and that should download the drawings. But at least based on this output, everything seems to work correctly.

**Comment by muesli** [comment]
There actually are drawings on the device. Also "0%, unknown" doesn't seem right either?

**Comment by muesli** [comment]
Also just noted: every time I restart tuhi-kete it claims this device hasn't been registered before, again.

**Comment by whot** [comment]
can you run tuhi with --verbose please? that should show the full exchange and provide a hint of what's going on here.

> Also "0%, unknown" doesn't seem right either?

That's a bug, but expected. we don't ask for battery info during the register process but we do on the first connect after that.

**Comment by muesli** [comment]
Ok, I managed to make all available drawings show up by drawing on a new page and pressing the button on the device:

```
        Battery level: 77%, discharging
        Available drawings:
                * 1562977254: drawn on the 2019-07-13 at 02:20
                * 1562979411: drawn on the 2019-07-13 at 02:56
                * 1563113734: drawn on the 2019-07-14 at 16:15
                * 1563114001: drawn on the 2019-07-14 at 16:20
```

Now the battery information is showing up, too!

We should probably update the README: it still mentions connecting via bluetoothctl which seems to be obsolete by now?

**Comment by whot** [comment]
yeah, true, those device notes are from well before we had a search functionality.

**Comment by whot** [comment]
> That's a bug, but expected.

Fixed in #139

**Comment by muesli** [comment]
Closing as I can't reproduce this anymore and probably just wasn't triggering "listen" correctly.
